src/excalidraw-app/collab/CollabWrapper.tsx

```diff
@@ -448,15 +448,8 @@ class CollabWrapper extends PureComponent<Props, CollabState> {
 
   private handleRemoteSceneUpdate = (
     elements: ReconciledElements,
-    {
-      init = false,
-      initFromSnapshot = false,
```

Comment:
I can't find any reference of initFromSnapshot in the codebase and this is a private method